### ISSUE TITLE
2019-12-06 M2SS_1TAUTOLD - remove on parameter at a time from PARAMETERLIST to identify sensitive parameters for which proper dependency management is not occurring leading to crashes

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#90]
## M2SS from 2019-12-06_M2SS_1TAUTOLD.json

```
 mct$PARAMETERLIST <- paste(c("AUCALL","AUCALLDN","AUCDN","AUCINFO","AUCINFOC","AUCINFODN","AUCINFOi","AUCINFP","AUCINFPC","AUCINFPDN","AUCINFPi","AUCLAST","AUCLASTi","AUCLASTC","AUCLASTCi","AUCLASTDN","AUCLASTDNi","AUCT","AUCT1_T2","AUCTAUDNi","AUCTAU","AUCTAUi","AUCTDN","AUCXPCTO","AUCXPCTOi","AUCXPCTP","AUCXPCTPi","AUCXBPCTO","AUCXBPCTP","AUMCINFOi","AUMCINFPi","AUMCLASTi","AUMCTAUi","AUMCXPTOi","AUMCXPTPi","C0","CAVi","CEST","CLAST","CLASTi","CLTAUi","CLTAUWi","CMAX","CMAXi","CMAXC","CMAXCi","CMAXDN","CMAXDNi","CMIN","CMINi","CMINDN","CMINDNi","CTROUGHi","CTROUGHENDi","DIi","DOSE","DOSEi","DOSEC","DOSECi","KEL","KELC0","KELNOPT","KELR","KELRSQ","KELRSQA","KELTMHI","KELTMLO","LASTTIME","LASTTIMEi","MRTIVIFO","MRTIVIFOi","MRTIVIFP","MRTIVIFPi","MRTLAST","MRTLASTi","PTF","PTFi","PTR","PTRi","PTROUGHRi","PTROUGHRENDi","TAU","TAUi","THALF","THALFF","TLAST","TLASTi","TMAX","TMAXi","TMIN","TMINi","TOLD","TOLDi","V0","VSSPi","VSSPWi","VSSOW","VSSOi","VSSOWi","FLGACCEPTPREDOSE","FLGACCEPTTMAX","FLGACCEPTKEL","FLGACCEPTTAU"), collapse=";")
```
## i:  AUCTAUi  removed
```
[1] "AUCTDN;AUCXPCTO;AUCXPCTOi;AUCXPCTP;AUCXPCTPi;AUCXBPCTO;AUCXBPCTP;AUMCINFOi;AUMCINFPi;AUMCLASTi;AUMCTAUi;AUMCXPTOi;AUMCXPTPi;C0;CAVi;CEST;CLAST;CLASTi;CLTAUi;CLTAUWi;CMAX;CMAXi;CMAXC;CMAXCi;CMAXDN;CMAXDNi;CMIN;CMINi;CMINDN;CMINDNi;CTROUGHi;CTROUGHENDi;DIi;DOSE;DOSEi;DOSEC;DOSECi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTIVIFO;MRTIVIFOi;MRTIVIFP;MRTIVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;TAU;TAUi;THALF;THALFF;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU"
unit_conversion : Unit Class 1 (Time) time_col:  TMAX TMAX1 TMIN TMIN1 TLAST TLAST1 KELTMLO KELTMHI THALF LASTTIME LASTTIME1 MRTLAST MRTLAST1 MRTIVIFO1 MRTIVIFP1 TAU1 TOLD1  parameters are scaled from  HR  to  HR  via scaling factor:  1 
unit_conversion : Unit Class 3 (Dose) dose_col:  DOSE1  parameters are scaled from  MG  to  MG  via scaling factor:  1 
unit_conversion : Unit Class 4 (Volume) volume_col:  VSSO1 VSSP1 VSSOW1 VSSPW1  parameters are scaled from  L  to  L  via scaling factor:  1 
unit_conversion : Unit Class 5 (Amount/Volume) conc_col:  C0 CMAX CMAX1 CMAXC CMAXC1 CMIN CMIN1 CLAST CLAST1 CTROUGH1 CTROUGHEND1 KELC0 CEST CAV1  parameters are scaled from  MG/L  to  NG/ML  via scaling factor:  1000 
unit_conversion : Unit Class 6 (1/Time) kel_col:  KEL  parameters are scaled from  1/HR  to  1/HR  via scaling factor:  1 
unit_conversion : Unit Class 7 (Volume/Time) cl_col:  CLTAU1  parameters are scaled from  L/HR  to  L/HR  via scaling factor:  1 
Error in unit_conversion(data = data_data, map = map_data, result = computation_df,  : 
  object 'auc_unit_tmp' not found
In addition: There were 50 or more warnings (use warnings() to see the first 50)
```
## i:  CLAST  removed
```
[1] "CLASTi;CLTAUi;CLTAUWi;CMAX;CMAXi;CMAXC;CMAXCi;CMAXDN;CMAXDNi;CMIN;CMINi;CMINDN;CMINDNi;CTROUGHi;CTROUGHENDi;DIi;DOSE;DOSEi;DOSEC;DOSECi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTIVIFO;MRTIVIFOi;MRTIVIFP;MRTIVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;TAU;TAUi;THALF;THALFF;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi"
Error in value[[3L]](cond) : 
  Error in auc_inf_o(conc = tmp_df[, map_data$CONC], time = tmp_df[, map_data$TIME], : object 'c_last' not found
For SDEID 50896291
In addition: There were 50 or more warnings (use warnings() to see the first 50)
```
## i:  CLTAUi  removed
```
[1] "CLTAUWi;CMAX;CMAXi;CMAXC;CMAXCi;CMAXDN;CMAXDNi;CMIN;CMINi;CMINDN;CMINDNi;CTROUGHi;CTROUGHENDi;DIi;DOSE;DOSEi;DOSEC;DOSECi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTIVIFO;MRTIVIFOi;MRTIVIFP;MRTIVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;TAU;TAUi;THALF;THALFF;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST"
Error in value[[3L]](cond) : 
  Error in cltauw(cltau = cl_tau[[d]], normbs = norm_bs): object 'cl_tau' not found
For SDEID 50896291
In addition: Warning messages:
1: In run_M2_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTKELCRIT' values provided via 'map' does not have a valid parameter name 'AUCXPCTP'
2: In run_M2_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGEMESIS' is not present in the dataset
3: In run_M2_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTKELCRIT' values provided via 'map' does not have a parameter name that is generated as an output 'KELRSQ'Flag 'FLGACCEPTKELCRIT' values provided via 'map' does not have a parameter name that is generated as an output 'KELNOPT'
4: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'AMOUNTU' #2# is not present in the dataset provided via 'map'
5: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'PKPTMU', 'CONCU' and 'DOSEU' #8# are not present in the dataset provided via 'map'
6: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'PKPTMU' and 'CONCU' #10# are not present in the dataset provided via 'map'
7: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'CONCU' and 'NORMBSU' #14# are not present in the dataset provided via 'map'
8: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'CONCU', 'PKPTMU and 'NORMBSU' #19# are not present in the dataset provided via 'map'
9: In run_M2_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTKELCRIT' values provided via 'map' does not have a valid parameter name 'AUCXPCTP'
10: In run_M2_SS_computation(data = merged_data, map = map_data, method = method,  :
 
 Show Traceback
 
 Rerun with Debug
 Error in value[[3L]](cond) : 
  Error in cltauw(cltau = cl_tau[[d]], normbs = norm_bs): object 'cl_tau' not found
For SDEID 50896291 
```

## i:  CTROUGHi  removed
```
[1] "CTROUGHENDi;DIi;DOSE;DOSEi;DOSEC;DOSECi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTIVIFO;MRTIVIFOi;MRTIVIFP;MRTIVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;TAU;TAUi;THALF;THALFF;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CLTAUi"
Error in value[[3L]](cond) : 
  Error in ptroughr(cmax = c_maxi[[d]], ctrough = c_troughi[[d]]): object 'c_troughi' not found
For SDEID 50896291
In addition: There were 50 or more warnings (use warnings() to see the first 50)

```

## i:  CTROUGHENDi  removed
```
[1] "DIi;DOSE;DOSEi;DOSEC;DOSECi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTIVIFO;MRTIVIFOi;MRTIVIFP;MRTIVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;TAU;TAUi;THALF;THALFF;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CLTAUi;CTROUGHi"
Error in value[[3L]](cond) : 
  Error in ptroughrend(cmax = c_maxi[[d]], ctrough = c_troughendi[[d]]): object 'c_troughendi' not found
For SDEID 50896291
In addition: Warning messages:
1: In run_M2_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTKELCRIT' values provided via 'map' does not have a valid parameter name 'AUCXPCTP'
2: In run_M2_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGEMESIS' is not present in the dataset
3: In run_M2_SS_computation(data = merged_data, map = map_data, method = method,  :
 
 Show Traceback
 
 Rerun with Debug
 Error in value[[3L]](cond) : 
  Error in ptroughrend(cmax = c_maxi[[d]], ctrough = c_troughendi[[d]]): object 'c_troughendi' not found
For SDEID 50896291 
```

## i:  KEL  removed
```
[1] "KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTIVIFO;MRTIVIFOi;MRTIVIFP;MRTIVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;TAU;TAUi;THALF;THALFF;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CLTAUi;CTROUGHi;CTROUGHENDi"
Error in value[[3L]](cond) : 
  Error in matrix(ncol = length(elist), nrow = 0): object 'elist' not found
For SDEID 50896291
In addition: There were 50 or more warnings (use warnings() to see the first 50)
```

## i:  MRTIVIFOi  removed
```
[1] "MRTIVIFP;MRTIVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;TAU;TAUi;THALF;THALFF;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CLTAUi;CTROUGHi;CTROUGHENDi;KEL"
Error in value[[3L]](cond) : 
  Error in auc_XpctO(conc = tmp_df[, map_data$CONC], time = tmp_df[, map_data$TIME], : object 'aucinf_o' not found
For SDEID 50896291
In addition: There were 50 or more warnings (use warnings() to see the first 50)
```

## i:  MRTLASTi  removed
```
[1] "PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;TAU;TAUi;THALF;THALFF;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CLTAUi;CTROUGHi;CTROUGHENDi;KEL;MRTIVIFOi"
Error in value[[3L]](cond) : 
  Error in auc_inf_o(conc = tmp_df[, map_data$CONC], time = tmp_df[, map_data$TIME], : object 'auclast' not found
For SDEID 50896291
In addition: There were 41 warnings (use warnings() to see them)
```

## i:  TLAST  removed
```
[1] "TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CLTAUi;CTROUGHi;CTROUGHENDi;KEL;MRTIVIFOi;MRTLASTi"
Error in value[[3L]](cond) : 
  Error in auc_last(conc = tmp_df[, map_data$CONC], time = tmp_df[, map_data$TIME], : object 't_last' not found
For SDEID 50896291
In addition: There were 50 or more warnings (use warnings() to see the first 50)
```

## i:  VSSPi  removed
```
[1] "VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CLTAUi;CTROUGHi;CTROUGHENDi;KEL;MRTIVIFOi;MRTLASTi;TLAST"
Error in value[[3L]](cond) : 
  Error in vss(cl = cl_tau[[d]], mrt = mrtivifpi[[d]]): object 'mrtivifpi' not found
For SDEID 50896291
In addition: There were 50 or more warnings (use warnings() to see the first 50)
```

## i:  AUCTAUi  removed
```
[1] "CLAST;CLTAUi;CTROUGHi;CTROUGHENDi;KEL;MRTIVIFOi;MRTLASTi;TLAST;VSSPi"
Error in value[[3L]](cond) : 
  Error in auc_last(conc = tmp_df[, map_data$CONC], time = tmp_df[, map_data$TIME], : object 't_max' not found
For SDEID 50896291
In addition: There were 50 or more warnings (use warnings() to see the first 50)
```

**Comment by jhhughes256** [comment]
### Update

- [ ] AUCTAUi removed "AUCTDN;AUCXPCTO;AUCXPCTOi;AUCXPCTP;AUCXPCTPi;AUCXBPCTO;AUCXBPCTP;AUMCINFOi;AUMCINFPi;AUMCLASTi;AUMCTAUi;AUMCXPTOi;AUMCXPTPi;C0;CAVi;CEST;CLAST;CLASTi;CLTAUi;CLTAUWi;CMAX;CMAXi;CMAXC;CMAXCi;CMAXDN;CMAXDNi;CMIN;CMINi;CMINDN;CMINDNi;CTROUGHi;CTROUGHENDi;DIi;DOSE;DOSEi;DOSEC;DOSECi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTIVIFO;MRTIVIFOi;MRTIVIFP;MRTIVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;TAU;TAUi;THALF;THALFF;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU"
- [ ] CLAST removed "CLASTi;CLTAUi;CLTAUWi;CMAX;CMAXi;CMAXC;CMAXCi;CMAXDN;CMAXDNi;CMIN;CMINi;CMINDN;CMINDNi;CTROUGHi;CTROUGHENDi;DIi;DOSE;DOSEi;DOSEC;DOSECi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTIVIFO;MRTIVIFOi;MRTIVIFP;MRTIVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;TAU;TAUi;THALF;THALFF;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi"
- [ ] CLTAUi removed "CLTAUWi;CMAX;CMAXi;CMAXC;CMAXCi;CMAXDN;CMAXDNi;CMIN;CMINi;CMINDN;CMINDNi;CTROUGHi;CTROUGHENDi;DIi;DOSE;DOSEi;DOSEC;DOSECi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTIVIFO;MRTIVIFOi;MRTIVIFP;MRTIVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;TAU;TAUi;THALF;THALFF;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST"
- [ ] CTROUGHi removed "CTROUGHENDi;DIi;DOSE;DOSEi;DOSEC;DOSECi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTIVIFO;MRTIVIFOi;MRTIVIFP;MRTIVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;TAU;TAUi;THALF;THALFF;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CLTAUi"
- [ ] CTROUGHENDi removed "DIi;DOSE;DOSEi;DOSEC;DOSECi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTIVIFO;MRTIVIFOi;MRTIVIFP;MRTIVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;TAU;TAUi;THALF;THALFF;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CLTAUi;CTROUGHi"
- [ ] KEL removed "KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTIVIFO;MRTIVIFOi;MRTIVIFP;MRTIVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;TAU;TAUi;THALF;THALFF;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CLTAUi;CTROUGHi;CTROUGHENDi"
- [ ] MRTIVIFOi removed "MRTIVIFP;MRTIVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;TAU;TAUi;THALF;THALFF;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CLTAUi;CTROUGHi;CTROUGHENDi;KEL"
- [ ] MRTLASTi removed "PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;TAU;TAUi;THALF;THALFF;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CLTAUi;CTROUGHi;CTROUGHENDi;KEL;MRTIVIFOi"
- [ ] TLAST removed "TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;V0;VSSPi;VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CLTAUi;CTROUGHi;CTROUGHENDi;KEL;MRTIVIFOi;MRTLASTi"
- [ ] VSSPi removed "VSSPWi;VSSOW;VSSOi;VSSOWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CLTAUi;CTROUGHi;CTROUGHENDi;KEL;MRTIVIFOi;MRTLASTi;TLAST"
- [ ] AUCTAUi removed "CLAST;CLTAUi;CTROUGHi;CTROUGHENDi;KEL;MRTIVIFOi;MRTLASTi;TLAST;VSSPi"

**Comment by rudraya** [comment]
This issue was addressed in a377a56, please let me know if it still occurs. I tested again using the parameters list provided in the above comment for when:
- AUCTAUi removed
- CLTAUi removed
- KEL removed

**Comment by jhhughes256** [comment]
### Update

As of a377a56 all test cases listed here run without computation error.
